indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -78,14 +78,26 @@
                                     {% trans %}n/a{% endtrans %}
                                 {% endif %}
                             </td>
-                            <td class="i-table">
-                                <select>
-                                    <option>--No session--</option>
-                                    {% for section in sections %}
-                                        <option>{{ section.name }}</option>
-                                    {% endfor %}
-                                    <option>Add to new session...</option>
-                                </select>
+                            <td class="i-table" style="white-space: normal;">
+                                {%- set text_color = '#' + contrib.session.colors.text -%}
```

Comment:
I wonder if it would make sense to have an `as_css` (or just `css`?) property on the `ColorTuple` class which returns 'color: ... !important; background-color: ... !important;'

That way you could just use `style="{{ contrib.session.colors.as_css }}"`.

In how many places to we use the color like this?